content/docs/about/policies/httplisteneroption.md

```diff
@@ -90,12 +90,12 @@ spec:
 
 ## Conflicting policies
 
-If you create multiple HttpListenerOption resources and attach them to the same gateway listener by using the `targetRefs` option, only the HttpListenerOption that was first created is applied. 
+If you create multiple HttpListenerPolicy resources and attach them to the same gateway listener by using the `targetRefs` option, only the HttpListenerPolicy that was first created is applied. 
 
 {{% callout type="info" %}}
-You cannot attach multiple HttpListenerOption resources to the same listener, *even if* they define different top-level policies. To add multiple policies, define them in the same HttpListenerOption resource.
+You cannot attach multiple HttpListenerPolicy resources to the same listener, *even if* they define different top-level policies. To add multiple policies, define them in the same HttpListenerPolicy resource.
```

Comment:
we should clarify how conflicting resources work with these new resources as this description assumes that you can specify multiple targetRefs. in our case, there is only a targetRef and I am wondering if they need to define all policies in the same httplistenerpolicy. because if they don't then they also can't create multiple httplistenerpolicy resources and attach them to the same gateway listener.